option optimize_for = LITE_RUNTIME;
message SuccInfo {
	required int32 id = 1;
	required float cp = 2;
	optional bool isNext = 3;
}

message Node {
	required int32 id = 1;
	required uint64 uniqKey = 2;
	repeated uint64 keys = 3;
	repeated SuccInfo succinfo = 4;
}

message Perception {
	required int32 nodeNum = 1;
	repeated Node nodes = 2;
}
